Add 2019-06-26 all hands meeting (#60)
hyphacoop/organizing: 2019-06-26-all-hands-meeting.md

## Notes

- Finance - Waiting on articles
- Business Planning - No Update
- Governance - No Update
- Infra & Ops
    - Review of infra list
        - Garry suggested a CalDev "thing" (Google Calendar)
        - Garry floated OwnCloud/NextCloud as a thing for file syncing
    - Meeting Friday
- PM role convo: https://hackmd.io/@patcon/ryMxS3okB/https%3A%2F%2Fhackmd.io%2FrJknEG_BS_emlrsU_qvcZQ%3Fview?type=book
    - PatCon - Org level pm to push initiative forward
    - [Questions](https://hackmd.io/@patcon/ryMxS3okB/https%3A%2F%2Fhackmd.io%2FrJknEG_BS_emlrsU_qvcZQ%3Fview?type=book) in pad
    - Distribute PM roll across two people due to bandwith
    - Re-align to Initiatves vs WG
        - Multidiciplanary team
        - Org or Project level
        - Clearly scope to org level
            - "Want want to do X" vs "This is a infra"
    - PM roll cultivate feedback
        - Sprints
        - Retros
        - Build team effectiveness
    - Develop kadence in team
        - Challange of asyncronus
    - Encouraging empathy and gratitude

>     - Follow throught (help define)

Reviewer: ```suggestion
    - Follow through (help define)
```

        - Clearly defined start of project
        - Clearly defined end of project
        - Deadlines :thumbsup: 
    - Trello
        - When trialing software
            - Not written in stone
            - Can you export/Escape hatch 
            - "exit" plan 
        - Kanban style
        - Predictability
        - Start and End date of trial 
            - retrospective
            - did it work
            - did it not
            - Why
        - Review next call
        - Not disolve working group 
            - Clear idea what we need to be tackling in next.... months  (4-5)

>             - 

Reviewer: ```suggestion
```

    - Other's experience (Yurko):
        - reflecting on his exprieence (organized chaos)
        - wild-west of development
        - Nice to work in a less chaotic way
        - User research is a challenge
        - We (Yurko's workplace) tried a number of techiques (via distributed team)
        - Resorted to a physical kaban for a project
        - Do more video calls
        - Regional managers meet 4 times a year

- Contracts and money meeintg 
    - https://hackmd.io/okYnBNQvSey-GcbfOI3evw

> - Linked in as Branding

Reviewer: ```suggestion
- LinkedIn as Branding
```

    - We dont have a website

>     - Linked is effort

Reviewer: ```suggestion
    - LinkedIn is effort
```

        - Who we are
        - What we done
    - Possibly forcing people onto it
    - Do we want this thing?
    - Do we tell people who we are some how else
    - Is it ethicly not where we want to be
    - Purpose

>         -  Cant advocate without a presense

Reviewer: ```suggestion
        -  Can't advocate without a presence
```

>         -  Would entities consier workign with a group like this

Reviewer: ```suggestion
        -  Would entities consider working with a group like this
```

>     -  Do we want to be know?

Reviewer: ```suggestion
    -  Do we want to be known?
```

>         -  Yurko mentioend that currenty Aether/Burak is low risk and good test

Reviewer: ```suggestion
        -  Yurko mentioned that currently Aether/Burak is low risk and good test
```

>         -  Lead is genreation important?

Reviewer: ```suggestion
        -  Lead is generation important?
```

    -  Pat: Value lead generation

>         -  is there enough bandwtih

Reviewer: ```suggestion
        -  is there enough bandwidth
```

        -  bring in contractors?
    -  Garry: start the converstion
        -  Transparent

- Decision for meeting format 
    - Every two weeks week
        - 15 min Stand up
    - Every month formal
    - +1 Elon Gary Yurko
    
## Follow Up

- Org PM role (patcon, garry)

### Actionable Items
- [ ] Review past #todos and recover potentials lost items

> - [ ] 

Reviewer: ```suggestion
```

- [ ] Consensus by team members and cooperative about how to split $
    - [ ] All: Research ahead of next meeting
    - [ ] All: Discuss this item at next all-hands
- [ ] Ben wants to on our next call try equal speaking time
- [ ] All: reflect on how we can interact in ways that make things move forward (e.g. forcing each other to express how they feel as to make others feel more permissioned to push things forward more assertively)

> 

Reviewer: ```suggestion
```

- [ ] create thread for discussion of lead generation

### Upcoming Discussions

> https://hackmd.io/gHIERg6JQ3m-igLd_1jPeQ?both

Reviewer: ```suggestion
```

> - Linked in as Branding

Reviewer: What is this for?
